site/docs/4.1/assets/js/src/application.js

```diff
@@ -58,6 +58,16 @@
       $(this).siblings('.progress').find('.progress-bar-striped').toggleClass('progress-bar-animated')
     })
 
+    // Add Toggle Spinner buttons
+    $('.bd-example-spinner').each(function (i, el) {
+      var btnHtml = '<button type="button" class="btn btn-secondary btn-spinner" data-toggle="button" aria-pressed="false" autocomplete="off">Toggle Spinner</button>'
```

Comment:
this line should be before the call of `.each`

Reply:
I'm not that familiar with the js best practices, but wouldn't it be better to use something like `document.createElement` here?

Reply:
@MartijnCuppens it depends if you work on the created element or not.
If you work on the created element (for example by adding attributes) you should use `document.createElement` if not you can use `innerHTML` or `appendChild`

The issue here, it's `btnHtml` is created on each iteration instead of one